test-suite-http-server-tck-netty/build.gradle

```diff
@@ -48,7 +48,7 @@ graalvmNative {
         enabled = true
     }
     binaries {
-        all {
+        configureEach {
```

Comment:
Same here, `configureEach` doesn't make sense since it's a model element, so it will register tasks in any case. This will never be lazy.

Reply:
reverted